regolith/helpers/l_contactshelper.py

```diff
@@ -0,0 +1,183 @@
+"""Helper for finding and listing contacts.
+
+   Projecta are small bite-sized project quanta that typically will result in
+   one manuscript.
+"""
+import datetime as dt
+import dateutil.parser as date_parser
+from dateutil.relativedelta import relativedelta
+import sys
+
+from pathlib import Path
+from regolith.dates import get_due_date
+from regolith.helpers.basehelper import SoutHelperBase
+from regolith.fsclient import _id_key
+from regolith.tools import (
+    all_docs_from_collection,
+    get_pi_id,
+)
+
+TARGET_COLL = "contacts"
+HELPER_TARGET = "l_contacts"
+
+THIS_DIR = Path('.')
+inst_file = THIS_DIR / ".." / ".." / ".." / ".." / "dbs" / "rg-db-public" / "db" / "institutions.yml"
+
+def subparser(subpi):
+    subpi.add_argument("-n", "--name", help='match contacts to a name fragment')
+    subpi.add_argument("-i", "--inst", help='match people to institutions')
+    subpi.add_argument("-d", "--day", help='match people to days')
+    subpi.add_argument("-m", "--month", help='match people to months')
+    subpi.add_argument("-y", "--year", help='match people to years')
+    subpi.add_argument("-r", "--misc", help='match people to miscellaneous notes')
+    '''
+    subpi.add_argument("-v", "--verbose", action="store_true", help='increase verbosity of output')
```

Comment:
since we are only ever returning one contact, I don;t think we need a verbose setting